stellar/deploy-contract.js

```diff
@@ -91,6 +102,15 @@ async function deploy(options, config, chain, contractName) {
     );
     const wasmHash = await uploadWasm(wasmPath, wallet, chain);
 
+    if (contractName === 'interchain_token') {
+        chain.contracts[contractName] = {
+            deployer: wallet.publicKey(),
+            wasmHash: serializeValue(wasmHash),
+        };
+        printInfo(contractName + ' Wasm Hash', serializeValue(wasmHash));
```

Comment:
the full contract config can be printed as JSON at the end of deployment

Reply:
updated. it will look like:
```
interchain_token: {
  "deployer": "GDEXMXS37JAZ54R2OEX56FV7OLYAOPIL7JH5QK6JEAOI3D2QALZT2KEI",
  "wasmHash": "e6598bf68a24110a075314d7564f9c6a677a48863f0c235e0d780686b7ca1cb8"
}
```